scripts/generate_maps.py

```diff
@@ -0,0 +1,177 @@
+import geopandas as gpd
+import pandas as pd
+import os
+import fiona
+from matplotlib_scalebar.scalebar import ScaleBar
+import matplotlib.pyplot as plt
+from matplotlib.patches import Patch
+
+# Define the paths
+input_file_path = "./antarctic_index.gpkg"
```

Comment:
The script will be more portable if you make these the input arguments.

I use the `argparse` library for this.